cypress/integration/e2e/header-control-menu/return-to-now-button.spec.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+describe("'Come back to now' button", () => {
+  context("when schedule is loaded", () => {
+    before(() => {
+      cy.loadScheduleToMonth();
+      cy.get("[data-cy=switch-next-month]").click();
+    });
+
+    context("when month is switched to next", () => {
```

Comment:
Nie jestem pewien co do tego czy ten `context` jest potrzebny.
Według mnie lepiej zrobić coś w stylu
```js
context("when schedule is loaded", () => {
      ...
      it("should return to current month from next month", () => {...})
      it("should return to current month from previous month" () => {...})
  }
```
@kopcion , co o tym myślisz?

Reply:
Sure, when it comes to e2e testing such granularity is not necessary.
Also instead of `'should return'` -> `'returns'`